app/models/micropost.rb

```diff
@@ -0,0 +1,16 @@
+class Micropost < ApplicationRecord
+  belongs_to :user
+  default_scope ->{order created_at: :desc}
+  mount_uploader :picture, PictureUploader
+  validates :user_id, presence: true
```

Comment:
```suggestion
  validates :user, presence: true
```
sự khác biệt giữa ` validates :user_id` và ` validates :user` là gì?

Reply:
em nghĩ là user nó có bao gồm user_id :v: